awx/settings/defaults.py

```diff
@@ -694,11 +694,6 @@ def IS_TESTING(argv=None):
     'cn-north-1': _('China (Beijing)'),
 }
 
-EC2_REGIONS_BLACKLIST = [
```

Comment:
@AlanCoding I don't actually see where or how these settings are enforced?  Has this just been broken for a long time?

Reply:
I thought these were converted to env variables and then passed to boto directly.

Reply:
Yes, Matt is right, this might actually have been dropped on the floor in the transition to plugins

https://github.com/ansible/awx/blob/9.3.0/awx/main/models/inventory.py#L2095

We used this for the old setting `regions_exclude` for the script ini config.

The plugin has the option `regions`, but no include / exclude separate settings.

So you could check this setting in the modern loop for the plugin and exclude them... but we still use the keyword "all" if user doesn't provide regions. But if the user provided one of these regions, would we really want to remove them anyway? Outright removing this list is probably best.